uiowa/uiowa
# People list links directly to site

Fixes #5722 
# To fix still:

- [x] Anonymous users are not being redirected as expected when they have the node's url
- [x] Need to reset checkbox value to 0 when user removes all websites (otherwise can get 'Visitors are being redirected' message when no websites are linked).

# How to test

- checkout branch and pull, sync a site (I've been using default - `ddev blt ds --site=default`)
- Create or edit existing person nodes, adding websites to a couple and checking the 'Link directly to website' checkbox for at least one.
- Create a page with the people list block.
- For the default people listing (/people) and your list block, click on the people. People with no website and people with website but unchecked box should send you to the regular person node. People with website and checked box should send you directly to the site you added. 
- If you leave the checkbox checked but remove the site(s), save, and check the lists again it should send you to the regular node also. 
- While logged in you can still access the person node for people who link directly to a site. Copy the link for one of these nodes and open in a private/incognito tab. It should redirect you to the site now instead.
- Site splits to test:
  - admissions.uiowa.edu
  - immuno.grad.uiowa.edu 
  - isa.uiowa.edu
  - now.uiowa.edu
  - pharmacy.uiowa.edu
  - sppa.uiowa.edu
  - uipress.uiowa.edu
- Feature splits to test:
  - event
  - grad_person_extended
